src/API/REST/V3/Routes/Donors/DonorController.php

```diff
@@ -0,0 +1,473 @@
+<?php
+
+namespace Give\API\REST\V3\Routes\Donors;
+
+use Give\API\REST\V3\Routes\CURIE;
+use Give\API\REST\V3\Routes\Donors\ValueObjects\DonorAnonymousMode;
+use Give\API\REST\V3\Routes\Donors\ValueObjects\DonorRoute;
+use Give\Donors\DonorsQuery;
+use Give\Donors\Models\Donor;
+use Give\Donors\ViewModels\DonorViewModel;
+use WP_Error;
+use WP_REST_Controller;
+use WP_REST_Request;
+use WP_REST_Response;
+use WP_REST_Server;
+
+/**
+ * The methods using snake case like register_routes() are present in the base class,
+ * and the methods using camel case like getSharedParams() are available only on this class.
+ *
+ * @unreleased Extends WP_REST_Controller class and rename methods
+ * @since 4.0.0
+ */
+class DonorController extends WP_REST_Controller
+{
+    /**
+     * @since 4.0.0
+     */
+    public function __construct()
+    {
+        $this->namespace = DonorRoute::NAMESPACE;
+        $this->rest_base = DonorRoute::BASE;
+    }
+
+    /**
+     * @since 4.0.0
+     */
+    public function register_routes()
+    {
+        register_rest_route($this->namespace, '/' . $this->rest_base, [
+            [
+                'methods' => WP_REST_Server::READABLE,
+                'callback' => [$this, 'get_items'],
+                'permission_callback' => [$this, 'get_items_permissions_check'],
+                'args' => array_merge($this->get_collection_params(), $this->getSharedParams()),
+                'schema' => [$this, 'get_item_schema'],
+            ],
+        ]);
+
+        register_rest_route($this->namespace, '/' . $this->rest_base . '/(?P<id>[\d]+)', [
+            [
+                'methods' => WP_REST_Server::READABLE,
+                'callback' => [$this, 'get_item'],
+                'permission_callback' => [$this, 'get_item_permissions_check'],
+                'args' => array_merge([
+                    'id' => [
+                        'description' => __('The donor ID.',
+                            'give'),
+                        'type' => 'integer',
+                        'required' => true,
+                    ],
+                    '_embed' => [
```

Comment:
@glaubersilva You shouldn't need to include this. It's internally supported by WP.